helm/charts/monitoring/templates/networkpolicy.yaml

```
@@ -0,0 +1,64 @@
+    ingress:
+        -   from:
+                -   podSelector:
+                        matchLabels:
+                            app.kubernetes.io/name: alloy
+                -   podSelector:
+                        matchLabels:
+                            app.kubernetes.io/name: grafana
+---
+# Grafana: Allow access via Ingress
+apiVersion: networking.k8s.io/v1
+kind:       NetworkPolicy
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)